Docker-Swarm-deployment/single-node/marketplace-apd/dmp-apd-stack.yaml

```diff
@@ -0,0 +1,43 @@
+version: '3.9'
+
+networks:
+  overlay-net:
+    external: true
+    driver: overlay
+
+services:
+  dmp-apd:
+    image: ghcr.io/datakaveri/dmp-apd-server-depl:1.0.0-alpha-8d09ca1
+    env_file:
+      - ./secrets/.apd.env
+    read_only: true
+    cap_drop:
+      - ALL
+    volumes:
+      - type: tmpfs
+        target: /tmp/
+        read_only: false
+    secrets:
+      - source: config
+        target: /usr/share/app/secrets/configs/config.json
+    restart: on-failure
+    networks:
+      - overlay-net
+    deploy:
+      replicas: 1
+      restart_policy:
+        condition: any
+        max_attempts: 5
+      placement:
+        constraints:
+          - "node.labels.marketplace-node==true"
+    logging:
```

Comment:
Add logging tags to each stack file to get container name and id as labels in logs:
``` 

 logging:
      driver: "json-file"
      options:
        max-file: "5"
        max-size: "10m"
        tag: "{\"name\":\"{{.Name}}\",\"id\":\"{{.ID}}\"}"
        
   ```
  because of this issue #639

Reply:
Done